src/libexec/zmrcd

```diff
@@ -100,6 +100,9 @@ my %REGEX_CHECKED_COMMANDS = (
    "rsync" => {
      regex => '[\'"a-zA-Z0-9\\/\\.\\-_:@= ]*',
      program => "rsync"},
+   "zmschedulesmpolicy" => {
+     regex => '[\'"e-g0-9\\-: ]*',
```

Comment:
why is this regex checking for specific e-g?

Reply:
We only have e, f, and g keywords to allow